dist/tpm2-abrmd.service.in

```diff
@@ -2,8 +2,9 @@
 Description=TPM2 Access Broker and Resource Management Daemon
 # These settings are needed when using the device TCTI. If the
 # TCP mssim is used then the settings should be commented out.
-After=dev-tpm0.device
-Requires=dev-tpm0.device
+After=dev-tpm0.device dev-tcm0.device
+ConditionPathExists=|/dev/tpm0
+ConditionPathExists=|/dev/tcm0
```

Comment:
Let me get an adult to look at this, I just pinged Lennart.

Reply:
Uh, no. This is wrong. After= and Requires= are orthogonal on systemd. That means that one will not imply the other, one is purely about ordering, and the other about pulling it into the transaction. But this means that dev-tpm0.device is never pulled into the initial transaction anymore after this PR (unless soemthing else happens to do that, but you cannot rely on that).

Hence this is not right. You drop the synchronization entirely here in the worst case, which doesn#t seem to be wat you want.

Reply:
or in other words, just replace 

```
After=dev-tpm0.device
Requires=dev-tpm0.device
```

by a simple:

```
After=tpm2.target
```

(Requires= is not needed there as the aforementioned systemd-tpm2-generator already pulls tpm2.target into the boot wherever necessary)

That said, this is v256 stuff. I have no good answer how you could cover this nicely on older systemd... v256 is still pretty new I guess.